// @mertushka/node-datachannel/dist/esm/lib/index.mjs
import nodeDataChannel from './node-datachannel.mjs';
import DataChannelStream$1 from './datachannel-stream.mjs';

function preload() {
  nodeDataChannel.preload();
}
function initLogger(level) {
  nodeDataChannel.initLogger(level);
}
function cleanup() {
  nodeDataChannel.cleanup();
}
function setSctpSettings(settings) {
  nodeDataChannel.setSctpSettings(settings);
}
function getLibraryVersion() {
  return nodeDataChannel.getLibraryVersion();
}
const Audio = nodeDataChannel.Audio;
const Video = nodeDataChannel.Video;
const Track = nodeDataChannel.Track;
const DataChannel = nodeDataChannel.DataChannel;
const PeerConnection = nodeDataChannel.PeerConnection;
const IceUdpMuxListener = nodeDataChannel.IceUdpMuxListener;
const RtcpReceivingSession = nodeDataChannel.RtcpReceivingSession;
const DataChannelStream = DataChannelStream$1;
var n = {
  initLogger,
  cleanup,
  preload,
  setSctpSettings,
  getLibraryVersion,
  RtcpReceivingSession,
  Track,
  Video,
  Audio,
  DataChannel,
  PeerConnection,
  DataChannelStream,
  IceUdpMuxListener
};

export { Audio, DataChannel, DataChannelStream, IceUdpMuxListener, PeerConnection, RtcpReceivingSession, Track, Video, cleanup, n as default, getLibraryVersion, initLogger, preload, setSctpSettings };
